website/src/utils/fontsourceUtils.ts

```diff
@@ -13,15 +13,14 @@ const fontsourceDownload = {
     };
   },
 
-  fontDownload(
-    fontId: string,
-    defSubset: string,
-    weight: number,
-    style: string
-  ) {
+  cssDownload(fontId: string, weight: number, style: string) {
     const dir = `${baseUrlDownload}/@fontsource/${fontId}`;
 
-    return `${dir}/files/${fontId}-${defSubset}-${weight}-${style}.woff2`;
+    return `${dir}/${weight}${style === "normal" ? "" : `-${style}`}.css`;
```

Comment:
Another comment here explaining the ternary operator (for example "If style is normal, only search for weight CSS. e.g. 400.css instead of 400-italic.css") would make the code much quicker to read and understand too in future reviews.